--- liblinear-2.49/python/liblinear/liblinear.py.orig	2022-08-11 09:26:18.000000000 +0200
+++ liblinear-2.49/python/liblinear/liblinear.py	2023-01-17 21:00:59.301392367 +0100
@@ -20,10 +20,7 @@ __all__ = ['liblinear', 'feature_node',
            'print_null']
 
 try:
-    dirname = path.dirname(path.abspath(__file__))
-    dynamic_lib_name = 'clib.cp*'
-    path_to_so = glob(path.join(dirname, dynamic_lib_name))[0]
-    liblinear = CDLL(path_to_so)
+    liblinear = CDLL("liblinear.so.6")
 except:
     try :
         if sys.platform == 'win32':
--- liblinear-2.47/python/setup.py.orig	2024-02-24 08:30:08.070312591 +0100
+++ liblinear-2.47/python/setup.py	2024-02-24 08:46:01.101816242 +0100
@@ -89,9 +89,6 @@ class CleanCommand(clean_cmd):
                 rmtree(f)
 
 def main():
-    if not path.exists(cpp_dir):
-        create_cpp_source()
-
     if not path.exists(license_file):
         copyfile(license_source, license_file)
 
@@ -110,11 +107,6 @@ def main():
         url="https://www.csie.ntu.edu.tw/~cjlin/liblinear",
         license=license_name,
         install_requires=["scipy"],
-        ext_modules=[
-            Extension(
-                "{}.{}".format(PACKAGE_DIR, dynamic_lib_name), **kwargs_for_extension
-            )
-        ],
         cmdclass={"clean": CleanCommand},
     )
 
