--- liblinear-2.47/Makefile.orig	2024-02-24 08:46:26.145013905 +0100
+++ liblinear-2.47/Makefile	2024-02-24 09:27:57.268185000 +0100
@@ -13,14 +13,17 @@ endif
 
 all: train predict
 
-lib: linear.o newton.o blas/blas.a
-	$(CXX) $(SHARED_LIB_FLAG) linear.o newton.o blas/blas.a -o liblinear.so.$(SHVER)
+lib: liblinear.so
 
-train: newton.o linear.o train.c blas/blas.a
-	$(CXX) $(CFLAGS) -o train train.c newton.o linear.o $(LIBS)
+liblinear.so: linear.o newton.o
+	$(CXX) $(SHARED_LIB_FLAG) linear.o newton.o -o liblinear.so.$(SHVER) $(LIBS) ; \
+	ln -sf liblinear.so.$(SHVER) liblinear.so
 
-predict: newton.o linear.o predict.c blas/blas.a
-	$(CXX) $(CFLAGS) -o predict predict.c newton.o linear.o $(LIBS)
+train: liblinear.so train.c
+	$(CXX) $(CFLAGS) -o train train.c -L. -llinear $(LIBS)
+
+predict: liblinear.so predict.c
+	$(CXX) $(CFLAGS) -o predict predict.c -L. -llinear $(LIBS)
 
 newton.o: newton.cpp newton.h
 	$(CXX) $(CFLAGS) -c -o newton.o newton.cpp
