--- liblinear-2.1/python/liblinear.py.orig	2015-09-27 00:03:39.000000000 +0200
+++ liblinear-2.1/python/liblinear.py	2017-01-28 23:38:33.691264075 +0100
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 from ctypes import *
 from ctypes.util import find_library
 from os import path
--- liblinear-2.1/python/liblinearutil.py.orig	2015-09-27 00:03:40.000000000 +0200
+++ liblinear-2.1/python/liblinearutil.py	2017-01-28 23:38:44.447930619 +0100
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 import os, sys
 sys.path = [os.path.dirname(os.path.abspath(__file__))] + sys.path
 from liblinear import *
--- liblinear-2.30/python/commonutil.py.orig	2019-03-20 18:28:22.000000000 +0100
+++ liblinear-2.30/python/commonutil.py	2019-12-25 13:22:28.729573432 +0100
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 from __future__ import print_function
 from array import array
 import sys
